src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -136,6 +136,7 @@ namespace Microsoft::Console::VirtualTerminal
             SetForegroundColor = 10,

```

Comment:
Any reason this file and the `cpp` for this class had the execute bits set on it? I'm sure it doesn't _actually matter, but it's certainly curious...

Reply:
I don't actually get your point here. Do you mean the enum element `CopyToClipboard` is not a proper name, and should be changed to `SetXXX`?

Reply:
Oh no I was talking about this:

![image](https://user-images.githubusercontent.com/18356694/81978342-00b8f600-95f1-11ea-977e-c42496adfe33.png)

I've never seen anything like `100644 → 100755` before, but they do look an awful lot like `chmod` bits

Reply:
This can be fixed with...

`git add --chmod -x xyz/OutputStateMachineEngine.hpp`

(once for every file that is changed, of course.)

Reply:
@DHowett-MSFT Thanks. @zadjii-msft Got what you meant.

That may be because I used vim to edit that file through terminal and some unexpected thing happened. Already fixed.